File: Assets/Scripts/SideTrigger.js
public var faceValue = 0;

public var quieto = false;
public var prevPosition;

public var dieComp;

function OnTriggerEnter( other : Collider ) {

	dieGameObject = GameObject.Find("Dice1");

	dieComp = dieGameObject.GetComponent("DieValue");

	dieComp.currentValue = 7 - faceValue;

	//Debug.LogError(dieValueComponent.currentValue);

}

function FixedUpdate(){
	dice = GameObject.Find("Dice1");
	
	if (prevPosition != dice.transform.position){
		prevPosition = dice.transform.position;
		quieto = false;
	} else if (!quieto){
		quieto = true;
		//GameObject.Find("pjCono").transform.position = GameSystem.Tiles[dieComp.currentValue].Position;
		//dieValueComponent = dice.GetComponent("DieValue");
		//dieValueComponent.currentValue = 7 - faceValue;
		//Debug.LogError("Quieto: " + (7 - faceValue));
		//Debug.LogError(dieComp.currentValue);
	}
}

File: Assets/Scripts/Triggers/Dice/SideTrigger.cs
using System;
using System.Collections;
using UnityEngine;
namespace AssemblyCSharp {
    public class SideTrigger : MonoBehaviour {

        public int faceValue;

        private GameObject Die;
        private DieValue DieValue;
        private bool TriggerActive;

        public SideTrigger() { }

        public void Awake() {
            Die = GameObject.Find("Dice1");
            DieValue = Die.GetComponent<DieValue>();
            TriggerActive = false;
        }

        public void OnTriggerEnter(Collider other) {
            Debug.LogError("Pisó el nº " + (7 - faceValue));
            DieValue.currentValue = (7 - faceValue);
            TriggerActive = true;
        }

        public void OnTriggerExit(Collider other) {
            TriggerActive = false;
        }

        public void FixedUpdate() {

            if (!TriggerActive)
                return;

            //--Si todavía se está moviendo el dado, retornar
            if (!IsQuiet(Die))
                return;
            Debug.LogError("Salio el nº " + (7 - faceValue));
            
            //--Aca remuevo el trigger para que no vuelva a saltar, pero cuando arrastra de nuevo o se resetea el dado, hay que volver a agregarlo
            this.enabled = false; 
            //GameSystem.Instance.NextPlayer.Move(dieComp.currentValue);
        }

        public bool IsQuiet(GameObject obj) {
            return obj.rigidbody.velocity == Vector3.zero && obj.rigidbody.angularVelocity == Vector3.zero;
        }
    }
}